rust/tests/model.rs

```diff
@@ -25,30 +25,81 @@ fn throw_data() {
 fn bad_arglength() {
     let (lib, data) = get_model("stdnormal");
     let model = Model::new(&lib, data, 42).unwrap();
-    let theta = vec![];
-    let mut grad = vec![];
+    let theta = [];
+    let mut grad = [];
     let _ = model.log_density_gradient(&theta[..], true, true, &mut grad[..]);
 }
 
 #[test]
 fn logp_gradient() {
     let (lib, data) = get_model("stdnormal");
     let model = Model::new(&lib, data, 42).unwrap();
-    let theta = vec![1f64];
-    let mut grad = vec![0f64];
+    let theta = [1f64];
+    let mut grad = [0f64];
     let logp = model
         .log_density_gradient(&theta[..], false, true, &mut grad[..])
         .unwrap();
     assert_ulps_eq!(logp, (2. * PI).sqrt().recip().ln() - 0.5);
     assert_ulps_eq!(grad[0], -1f64);
 }
 
+// Note: this test does not require download-bridgestan-src feature
+// but it assumes that bridgestan_src is on the home folder
+// one way to have that assumption met is by running the example or
+// the model_downloading_and_compiling test before running this test
+#[cfg(target_family = "unix")]
```

Comment:
Does this still require unix-only when it is run on its own due to the `[ignore]`?